src/rdb.c

```diff
@@ -3208,15 +3208,19 @@ void startLoadingFile(size_t size, char* filename, int rdbflags) {
 /* Refresh the absolute loading progress info */
 void loadingAbsProgress(off_t pos) {
     server.loading_loaded_bytes = pos;
-    if (server.stat_peak_memory < zmalloc_used_memory())
+    if (server.stat_peak_memory < zmalloc_used_memory()) {
         server.stat_peak_memory = zmalloc_used_memory();
+        server.stat_peak_memory_time = time(NULL);
```

Comment:
i think we can just use `server.unixtime;` instead calling `time(NULL)` every time